### PR TITLE
added homepage 

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**
Screenshot (18)

### CODE REVIEW

**Review by vivekgudipudi — APPROVED** [review]

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+.ecom-navbar{
+    display:flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+    padding:var(--s);
+}
+.landing-page-name{
+justify-content: flex-start;
+}
```
Comment (smkmallik) [inline homepage.css:13]: format this in the next change

File: homepage.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+.ecom-navpills{
+    display:flex;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content:space-between;
+}
+.ecom-search:focus{
+outline:none;
+}
```
Comment (smkmallik) [inline homepage.css:23]: format this as well

**Review by smkmallik — APPROVED** [review]
Changes seems okay. Format the CSS changes in the next changes. You can merge the changes

**Review by MurtazaBagwala07 — APPROVED** [review]
> Remove the commented code if not required
> Try to use REM instead px as it will help in responsiveness
> Rest code looks clean

**Comment by vivekgudipudi** [comment]
try to use rems rather than px, try to implement default sizes and use them as variables,
Nice UX

**Pull request merged**